### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by pouriaho** [nutgram/nutgram#215]
### Nutgram version

^3.7

### Describe the bug

Hi,

### This is my code:
```php
<?php

use App\Commands\StartCommand;
use SergiX44\Nutgram\Nutgram;
use SergiX44\Nutgram\RunningMode\Webhook;

$bot = new Nutgram('TOKEN');
$bot->setRunningMode(Webhook::class);
$bot->onCommand('start', StartCommand::class);
$bot->run();
```

### StartCommand class:
```php
<?php

namespace App\Commands;

use SergiX44\Nutgram\Nutgram;

class StartCommand {
    public function __invoke(Nutgram $bot, $param)
    {
        $bot->sendMessage('START');
    }
}
```

### The above code produces this error:
```txt
[08-Oct-2022 16:41:09 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to a member function setParameters() on bool in E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\Nutgram.php:292
Stack trace:
#0 E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\Nutgram.php(262): SergiX44\Nutgram\Nutgram->fireExceptionHandlerBy('EXCEPTION', Object(ArgumentCountError))
#1 E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\Nutgram.php(247): SergiX44\Nutgram\Nutgram->fireHandlers(Array)
#2 E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\RunningMode\Webhook.php(68): SergiX44\Nutgram\Nutgram->processUpdate(Object(SergiX44\Nutgram\Telegram\Types\Common\Update))
#3 E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\Nutgram.php(219): SergiX44\Nutgram\RunningMode\Webhook->processUpdates(Object(SergiX44\Nutgram\Nutgram))
#4 E:\Programs\xampp\htdocs\ph\ds\index.php(119): SergiX44\Nutgram\Nutgram->run()
#5 {main}
  thrown in E:\Programs\xampp\htdocs\ph\ds\vendor\nutgram\nutgram\src\Nutgram.php on line 292
```

**where is the problem from ?!**

### To Reproduce

....

### Expected behavior

Sending «START» to user.

**Comment by Lukasss93** [comment]
Please read the doc page: https://nutgram.dev/docs/usage/handlers#oncommand

TLDR:

Change to this:
```php
$bot->onCommand('start {parameter}', StartCommand::class);
```

or this:
```php
class StartCommand {
    public function __invoke(Nutgram $bot)
```
